src/main.ts

```diff
@@ -17,6 +17,11 @@ async function run() {
 
     if(version) {
       await rustup.install();
+      if (os.platform() !== 'darwin') {
```

Comment:
since this action installs rustup on darwin, darwin should already have the most recent version

Reply:
We should probably move this inside the `rustup.install()` call and avoid leaking implementation details.

Reply:
We should probably move this inside the `rustup.install()` call and avoid leaking implementation details.

Reply:
done